corehq/apps/hqwebapp/static/hqwebapp/spec/urllib_spec.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-env mocha */
+/* global $, sinon */
+
+describe('urllib', function() {
+    var urllib = hqImport('hqwebapp/js/urllib.js');
+    describe('getUrlParameterFromString', function() {
+        it('should return null when URL param missing', function() {
+            assert.equal(urllib.getUrlParameterFromString('asdf', '?limit=29'), null);
```

Comment:
`assert.equal` must use `==` or other comparison that treats `null` and `undefined` as equal. I can at least fix the test to be less confusing, even though it doesn't catch the difference